### PR TITLE
Wt32 s3 86 v integration

### PR DESCRIPTION
**Pull request by moinmoin-sh**

### CODE REVIEW

File: release_extra.py [hunk 1]
```diff
@@ -6,6 +6,7 @@
 
 # get the firmware name from the [firmware] section in platformio.ini
 firmware_name = config.get("firmware", "name")
+firmware_name = firmware_name.replace('\\\"', '')
```
Comment (sumnerboy12) [inline release_extra.py:9]: I know you mentioned this before, but what is the reason for this? I don't remember the specifics but it was a real pain trying to get the firmware name to pass thru as DEFINEs for use inside the firmware.

I would be very interested if you have found a way to tidy that up!

**Review by sumnerboy12 — APPROVED** [review]
Looks great Moin - goes to show how well this was written to start with - that you can make such a substantial change with so few code changes - nice work!

File: release_extra.py (hunk 1 above)
Comment (moinmoin-sh) [reply release_extra.py:9]: The reason for the escape quotes was this define for LVGL in platformio.ini
`  -DLV_MEM_CUSTOM_INCLUDE=\"esp32-hal-psram.h\"
`
I had to make sure that the quotes are passed thru to the place in LVGL code where the `#include ....` is put together depending on customisation. This was necessary for linux only what is used in the automated release build.
I guess there is a similar requirement for the firmware name and such to be processed in the tool chain and get finally right in your code.
Unfortunately the firmware name keeps the escape quotes and uses ugly paths during build
`Linking .pio/build/wt32-eth/\"OXRS-IO-TouchPanel-ESP32-FW\"_wt32-eth_v2.0.1.elf
`
With the above modification I remove the escaped quotes to have a clean firmware name for further use.

**Comment by moinmoin-sh** [comment]
Would be helpfull if you could find out how to add the merged binary to the release package before we merge.

**Comment by moinmoin-sh** [comment]
I added the WT32S3-86V...._full.bin to the release (as discussed earlier) and marked the release as prerelease for now so it doesn't show up on the repro main page.

**Pull request merged**